networking/ovn_kubernetes_network_provider/configuring-gateway-mode.adoc

```diff
@@ -0,0 +1,13 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="configuring-gateway-mode"]
+= Configuring gateway mode
+include::_attributes/common-attributes.adoc[]
+:context: configuring-gateway-mode
+
+toc::[]
+
+As a cluster administrator you can configure the `gatewayConfig` object to manage how external traffic leaves the cluster. You do so by setting the `routingViaHost` spec to `true` for local mode or `false` for local mode.
+
+In the shared mode traffic is forwarded from pods in the cluster to a shared centralized node where it exits the cluster. In the local mode, each node serves as its own gateway so that traffic exits from a pod through the node it is running on.
```

Comment:
Again, this is wrong or, at least, misleading. Traffic always goes out from the pod node (unless there is an egress IP in place). The difference is whether it traverses the host network stack or OVS sends it directly to the physical interface.

Reply:
reworded